en/django_admin/README.md

```diff
@@ -35,8 +35,8 @@ To log in, you need to create a *superuser* - a user account that has control ov
 When prompted, type your username (lowercase, no spaces), email address, and password. **Don't worry that you can't see the password you're typing in – that's how it's supposed to be.** Type it in and press `enter` to continue. The output should look like this (where the username and email should be your own ones):
 
 ```
-Username: admin
-Email address: admin@admin.com
+Username: ola
+Email address: ola@admin.com
```

Comment:
While you're at it, mind changing it to `ola@example.com`? That domain exists exactly for these cases (https://tools.ietf.org/html/rfc2606#section-3).

Reply:
Made the modification myself, waiting on tests to finish before merging.